pages/phases/assessments/reviewing-sme-assessments.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: article
-permalink: /hiring-phases/phone-assessment-interviews/reviewing-sme-assessments/
+permalink: /hiring-phases/assessments/reviewing-sme-assessments/
 section: hiring-phases
 sidenav: hiring-phases
-phase: phone-assessment-interviews
+phase: assessments
 sub-phase-order: 030
 title: Reviewing SME Assessments
-intro: As SMEs submit their interview assessments, check their work and email them with any necessary feedback.
+intro: As SMEs submit assessment feedback, check their work and provide necessary feedback.
```

Comment:
strange to use feedback twice in the sentence? maybe it's fine

Reply:
Edited to "intro: As SMEs submit their assessment reviews, check their work and provide necessary feedback." and added a few more edits here: https://github.com/usds/SME-QA-Website/pull/243/commits/99f87d25aaadb989299a23964806537d4a5900cb